template.distillery/PROJECT_NAME_handlers.eliomi

```diff
@@ -0,0 +1,59 @@
+(* This file was generated by Ocsigen-start.
+   Feel free to use it, modify it, and redistribute it as you wish. *)
+
```

Comment:
I rearranged `PROJECT_NAME_handlers.eliom[i]` because there were duplicated handlers due to the old `include Os_handlers`.
I also changed the signature of the client handlers by removing `Deriving_Json` as in the previous comment.